admin/templates/resource/create.html

```diff
@@ -14,7 +14,7 @@
         <div class="w-1/2 mb-4">
           <div class="flex flex-col mr-3">
             <label for={{ attribute.name }} class="uppercase text-gray-500 text-xs mb-1">{{ attribute.name | format_label }}</label>
-            {% if 'VARCHAR' in attribute.type %}
+            {% if attribute.type == 'VARCHAR' %}
```

Comment:
this might not be correct as there are attribute types such as VARCHAR(50), VARCHAR(100) etc. I believe that was the reason to search for substring instead of full comparison.